### DISCUSSION TITLE
getting Missing "./dist/app/composables/fetch" export in "nuxt" package` error

### DISCUSSION THREAD

**Discussion by lecneri** [nuxt/nuxt#16876] [Questions]
I'm trying to find a way to use baseUrl in useFetch, and I've tried this
```
//composables/useFetch.ts
import { Ref } from 'nuxt/dist/app/compat/capi';
import {  useFetch as theirFetch, UseFetchOptions } from 'nuxt/dist/app/composables/fetch';

export function useFetch<DataT>(
  url: string | Request | Ref<string | Request> | (() => string) | Request,
  options?: UseFetchOptions<DataT>
) {
  return theirFetch(
    url,
    {
      ...options,
      baseURL: 'http://localhost:8000'
    },
  );
}
```
but vites complains 
`[plugin:vite:import-analysis] Missing "./dist/app/composables/fetch" export in "nuxt" package`
So I wonder if it's something like [this](https://github.com/nuxt/framework/issues/6087)

**Comment by TheAlexLichter** [comment]
Imports are the issues I think. You shouldn't need them at all due to auto import.

**Comment by lecneri** [comment]
@manniL ,
how do you recommend to set baseURL on fetch across all application?
